src/Tizen.Network.Bluetooth/Tizen.Network.Bluetooth/BluetoothEventArgs.cs

```diff
@@ -955,6 +955,79 @@ public string Address
         }
     }
 
+    public class AvrcpControlConnChangedEventArgs : EventArgs
+    {
+        private bool _isConnected;
+        string _remoteAddress;
+        // Setting Values when Event is invoked
+        internal AvrcpControlConnChangedEventArgs(bool conn, string address)
+        {
+            _isConnected = conn;
+            _remoteAddress = address;
+        }
+
+        public bool IsConnected
```

Comment:
simple getter pattern. 
```suggestion
        public bool IsConnected => _isConnected;
```